### PR TITLE
Dockerize and add Kubernetes manifests

### PR DESCRIPTION
**Pull request by riccardomc**
This PR introduces simple containerization driven by a Makefile and manifests to deploy the service broker on kubernetes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 build/
 product/
 release/
+vendor/
```
Comment (a-robinson) [inline .gitignore]: `vendor/` is a legitimate part of the repo. It shouldn't be ignored.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+crdbsb*
```
Comment (a-robinson) [inline .gitignore]: Where are these files coming from? Are they from your local environment? Are they build artifacts? Something else?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.9-alpine AS build
```
Comment (a-robinson) [inline Dockerfile]: Any particular reason to use 1.9-alpine rather than 1.10-alpine?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,27 @@
+SOURCES=$(shell ls -1 *.go)
+IMAGE ?= riccardomc/crdbsb
```
Comment (a-robinson) [inline Makefile]: This doesn't seem very shareable -- I doubt others will want their images tagged with your username.

**Review by a-robinson — COMMENTED** [review]
Hi @riccardomc, a proper full review of this may have to wait a day or two. In the meantime, this will need some sort of docs/instructions on how and why to use it. Also, what sort of testing have you done on it?

File: .gitignore (hunk 2 above)
Comment (riccardomc) [reply .gitignore]: yes, they are the result of `make crdbsb` and `make crdbsb-static`.

File: Makefile (hunk 4 above)
Comment (riccardomc) [reply Makefile]: Fixed.

File: Dockerfile (hunk 3 above)
Comment (riccardomc) [reply Dockerfile]: Updated to 1-10.

File: .gitignore (hunk 1 above)
Comment (riccardomc) [reply .gitignore]: removed.

**Comment by riccardomc** [comment]
Hi @a-robinson, thanks for taking the time to look at this. I have added a section in the `README.md` on how to use the included `Makefile`.

I've also generalized the name of the docker instances and the configuration of the CockroachDB cluster a little bit.

**Review by a-robinson — COMMENTED** [review]
Very nice! I'd just request that you replace `crdb` with `cockroachdb` to match the naming style we use everywhere else. Otherwise this looks good to me.

**Comment by riccardomc** [comment]
All names aligned to style. ;)

**Comment by riccardomc** [comment]
Also, I retested the whole thing with updated minikube [0.28.2](https://github.com/kubernetes/minikube/releases/tag/v0.28.2) and published a [blog post](https://container-solutions.com/cloud-services-for-your-kubernetes-applications/) about this.

**Comment by a-robinson** [comment]
Very cool! Thanks for sharing the post and for making these improvements.

**Pull request merged**